packages/publisher/gcs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@electron-forge/publisher-gcs",
+  "version": "6.4.2",
+  "description": "Google Cloud Storage publisher for Electron Forge",
+  "repository": "https://github.com/electron/forge",
+  "author": "Evgeny Vlasenko",
+  "license": "MIT",
+  "main": "dist/PublisherGCS.js",
+  "typings": "dist/PublisherGCS.d.ts",
+  "devDependencies": {
+    "chai": "^4.3.3",
+    "mocha": "^9.0.1"
+  },
+  "engines": {
+    "node": ">= 14.17.5"
+  },
+  "dependencies": {
+    "@electron-forge/publisher-base": "6.4.2",
+    "@electron-forge/shared-types": "6.4.2",
+    "@google-cloud/storage": "^7.1.0",
+    "debug": "^4.3.1",
+    "google-auth-library": "^9.0.0"
```

Comment:
`google-auth-library` should be a dependency at the top level `package.json` as well, I think?